modules.nix

```
@@ -0,0 +1,44 @@
+{ pkgs, lib, config, ...}:
+let
+  toBG = { red, green, blue }:
+    "${toString(255.0 / red)}, "   +
+    "${toString(255.0 / green)}, " +
+    "${toString(255.0 / blue)})";
```

This logic isn't correct.

Should be:

```suggestion
    "${toString(red   / 255.0)}, "   +
    "${toString(green / 255.0)}, " +
    "${toString(blue  / 255.0)})";
```

```bash
nix-repl> 255 / 255.0
1

nix-repl> 200 / 255.0
0.784314

nix-repl> 150 / 255.0
0.588235

nix-repl> 100 / 255.0
0.392157

nix-repl> 1 / 255.0
0.00392157

nix-repl> 0 / 255.0
0  # I wasnt expecting this, but is ok for our case
```